.evergreen/javaConfig.bash

```diff
@@ -0,0 +1,16 @@
+# Java configurations for evergreen
+# Defaults to Java 11
+
+export JDK8="/opt/java/jdk8"
+export JDK11="/opt/java/jdk11"
+export JDK17="/opt/java/jdk17"
+
+if [ -d "$JDK11" ]; then
+  export JAVA_HOME=$JDK11
+fi
+
+export JAVA_VERSION=${JDK:-11}
```

Comment:
Which scripts define the JDK variable?

Reply:
Good catch, thats a bug. Should be JAVA_VERSION as set in .evg.yml